app/lib/synthetic_study_populator.rb

```diff
@@ -148,7 +147,7 @@ def self.add_files(study, study_config, study_folder, user)
     end
   end
 
-  def self.process_expression_file_params(study, file_info)
+  def self.process_expression_file_params(_study, file_info)
```

Comment:
`_study` seems unused in this method, so ideal to remove here and in calling code.

```suggestion
  def self.process_expression_file_params(file_info)
```